### ISSUE TITLE
Cannot complete table list for DESC

### ISSUE THREAD

**Issue by lvht** [dbcli/mycli#139]
After enter `desc`, we cannot get the table list for completation.

**Comment by kakiezhang** [comment]
you mean “desc table"?

**Comment by spacewander** [comment]
It looks like the problem of `sqlparse`. `sqlparse` doesn't treat `desc table` specially.
I have modified the source code to add this feature for myself:
```python
def suggest_based_on_last_token(token, text_before_cursor, full_text, identifier):
    if text_before_cursor.rpartition(" ")[0].lower() in ('explain', 'desc', 'describe'):
        return [{'type': 'keyword'}, {'type': 'table', 'schema': ''}]

    if isinstance(token, string_types):
```

This is a hacking way. I don't know if other people can accept it.

**Comment by lvht** [comment]
:+1: @kakiezhang

**Comment by amjith** [comment]
@spacewander Do you mind opening a PR with your change?

**Comment by amjith** [comment]
This now addressed by PR #141 from @spacewander. 

You can install directly from master to use the new feature. 

```
pip install -U https://github.com/dbcli/mycli/archive/master.zip
```

**Comment by lvht** [comment]
:+1: @amjith
